generator/app.go

```diff
@@ -84,12 +84,23 @@ type GithubTeams struct {
 	Description string
 }
 
+// Repo represents a specific repo or subdirectories owned by the group
+type Repo struct {
+	Name        string
+	Description string
+	URL         string
+	Owners      []string
+}
+
 // Subproject represents a specific subproject owned by the group
 type Subproject struct {
 	Name        string
 	Description string
+	Label       string
+	Repos       []Repo
 	Owners      []string
```

Comment:
So here's an example of how you could add the human info, while preventing it from falling out of data with the OWNERS files:
- change this to `OwnersFiles []string`
- add a field `Owners []Person`
- update app/generator to verify that the people listed in `Owners` are actually present in the `OwnersFiles` as `approvers`

You have to do that third step though.  If you don't, then you're left with two sources of truth as to who has the final say, and that's more confusing than the situation today.